ui/package.json

```diff
@@ -21,66 +21,68 @@
     "prepare": "(cd .. && ./ui/node_modules/.bin/husky install)"
   },
   "dependencies": {
-    "@apollo/client": "^3.6.9",
-    "@babel/preset-typescript": "^7.18.6",
-    "@headlessui/react": "^1.6.6",
-    "@types/geojson": "^7946.0.8",
-    "@types/jest": "^28.1.4",
-    "@types/mapbox-gl": "^2.7.3",
-    "@types/react": "^18.0.15",
-    "@types/react-dom": "^18.0.6",
-    "@types/react-helmet": "^6.1.5",
+    "@apollo/client": "~3.6.10",
```

Comment:
Thanks for pointing this out, but updating these dependencies requires significant code modification, so it will be done in the scope of another PR. Here I update only easily upgradable dependencies altogether.